yarn-project/pxe/src/pxe_oracle_interface/pxe_oracle_interface.test.ts

```diff
@@ -75,6 +75,11 @@ describe('PXEOracleInterface', () => {
     capsuleDataProvider = new CapsuleDataProvider(store);
     keyStore = new KeyStore(store);
     simulationProvider = new WASMSimulator();
+
+    // PXEOracleInterface.syncTaggedLogs(...) function syncs logs up to the block number up to which PXE synced. We set
+    // as sufficiently high sync number here to not interfere with the tests.
+    await setSyncedBlockNumber(100);
```

Comment:
This seems very fragile. Why do we even need to do this in the setup?

Reply:
Because we need the PXE to be synced to a block larger than the largest block in any of the logs such that we get delivered all the logs by default in the tests.

Reply:
Makes sense to define the constants. Did that in https://github.com/AztecProtocol/aztec-packages/pull/13152/commits/fe43da92095af7fb692a7f2b7f8f905258f01e5c

Reply:
But why do it so early? We've not even created the PXE yet. And how do we know that block 3 is enough?

Reply:
>But why do it so early?

No reason. Moved it to the end of the beforeEach function as I agree it makes more sense there.

> And how do we know that block 3 is enough?

See the comment above the call to setSyncedBlockNumber. That block corresponds to the block of the last emitted log (as set by `generateMockLogs` function).